agent/taskresource/envFiles/envfile.go

```diff
@@ -121,6 +127,7 @@ func (envfile *EnvironmentFileResource) Initialize(resourceFields *taskresource.
 	envfile.s3ClientCreator = factory.NewS3ClientCreator()
 	envfile.ioutil = ioutilwrapper.NewIOUtil()
 	envfile.bufio = bufiowrapper.NewBufio()
+	envfile.ipCompatibility = config.InstanceIPCompatibility
```

Comment:
`config.InstanceIPCompatibility ` is being queried in 2 places: here, and in task.go/InitializeResources(). Seems redundant

Reply:
synced offline, Initialize() is used during agent restarts while the other is used to initialize task resource when a task payload arrives.